tests/unit/lifecycle/CircuitBreakerGroup.spec.js

```diff
@@ -967,7 +967,7 @@ describe('extractBucketProcessorCircuitBreakerConfigs', () => {
                     transition: {
                         location: {},
                         topic: {},
-                        global: [ ],
+                        global: [],
```

Comment:
Please note that this change is just to fix a lint warning , will be separated in a specific commit post reviews